pyaedt/desktop.py

```diff
@@ -589,16 +591,16 @@ def _exception(self, ex_value, tb_data):
         """
         try:
             oproject = self._main.oDesktop.GetActiveProject()
-            proj_name = oproject.GetName()
+            project_name = oproject.GetName()
```

Comment:
@maxcapodi78 This variable is assigned but never used. We could get rid of it.

Reply:
I agree. that method can be refactored